include/rangers-api/Application/MyApplication.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+namespace app {
+    class MyApplication : public hh::game::GameApplication {
+        csl::ut::MoveArray<void*> unk201;
+    public:
+        MyApplication(csl::fnd::IAllocator* pAllocator, void* unkParam1);
+        static MyApplication* Instantiate(void* unkParam1, csl::fnd::IAllocator* pAllocator);
```

Comment:
Instantiate function name is never used in these games, these functions are more representative of Create functions in [LWAPI](https://github.com/hedge-dev/LWAPI/blob/9affbbfc3d30b81ca22bb8c76e6e13557e876d78/Sonic2013/Hedgehog/Game/GOComponent.h#L83).

Reply:
Pointer types should be identified by a `p` before the variable name.